packages/animations/test/open_container_test.dart

```diff
@@ -1733,6 +1733,46 @@ void main() {
       expect(modalRoute.settings, routeSettings);
     },
   );
+
+  testWidgets(
```

Comment:
Add a comment here to help with context:
```suggestion
    // Regression test for https://github.com/flutter/flutter/issues/72238.
    testWidgets(
```